proxies/live/apiproxy/targets/ig3.xml

```diff
@@ -16,6 +16,10 @@
         <Name>AssignMessage.SetAccessModeUserRestricted</Name>
         <Condition>(scope JavaRegex "(.+\ urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service\ .+|^urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service\ .+|.+\ urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service$|^urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service$|.+\ personal-demographics-service:USER-RESTRICTED\ .+|^personal-demographics-service:USER-RESTRICTED\ .+|.+\ personal-demographics-service:USER-RESTRICTED$|^personal-demographics-service:USER-RESTRICTED$)")</Condition>
       </Step>
+      <Step>
+        <Name>AssignMessage.SetAccessModeCitizenAccess</Name>
+        <Condition>(scope JavaRegex "(.+\ urn:nhsd:apim:user-nhs-login:P9:personal-demographics-service\ .+|^urn:nhsd:apim:user-nhs-login:P9:personal-demographics-service\ .+|.+\ urn:nhsd:apim:user-nhs-login:P9:personal-demographics-service$|^urn:nhsd:apim:user-nhs-login:P9:personal-demographics-service$)") and apim.feature_toggles.enable-citizen-access = true</Condition>
```

Comment:
If `apim.feature_toggles.enable-citizen-access != true` then we don't prevent citizen access, we just don't restrict it. The feature toggle isn't doing what we want here.